src/components/App.jsx

```diff
@@ -22,12 +28,17 @@ class App extends React.Component {
     });
```

Comment:
Your promise chain above (lines 23 & 26) should be indented inside your axios.get :)

Reply:
Ah thanks!